pytorch_lightning/trainer/properties.py

```diff
@@ -488,6 +489,16 @@ def train_loop(self) -> FitLoop:
         # FIXME(@awaelchli): the current train_loop should be renamed to fit_loop
         return self.fit_loop
 
+    @property
+    def evaluation_loop(self) -> EvaluationDataLoaderLoop:
+        if self.training or self.sanity_checking:
```

Comment:
Actually, we set `self.trainer.validating = True` before accessing this property so we do need:

```suggestion
        if self.state.fn == TrainerFn.FIT or self.sanity_checking:
```

Reply:
Good call !

Reply:
Done !